style.css

```diff
@@ -579,6 +581,135 @@ input[type="button"]:disabled {
   color: #67798e;
 }
 
+.card-works {
+  grid-template-rows: 82px 36px 220px 220px 56px;
+  background: #fff;
+  width: 343px;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  align-items: center;
+  visibility: hidden;
+  opacity: 0;
+  transition: visibility 0 opacity 0.5s;
+  margin-left: 16px;
+  margin-top: 60px;
+  margin-right: 16px;
+}
+
+.bg-active {
+  visibility: visible;
+  opacity: 1;
+}
+
+.top-desktop {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.top-desktop .heading-part h1 {
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  position: relative;
+  left: 16px;
+  top: 20px;
+  right: 14px;
+  display: block;
+  margin-left: 16px;
+}
+
+.heading-part h2 {
+  display: none;
+}
+
+.heading-part .fa-times {
+  position: absolute;
+  top: 40px;
+  left: 330px;
+  color: #67798b;
+  margin-right: 100px;
+}
+
+.technologies {
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+  margin-top: 52px;
+}
+
+.snapshot {
+  float: none;
+  margin-left: 16px;
+  margin-top: 24px;
+}
+
+.technologies .tec-list {
+  display: inline;
+}
+
+.description {
+  margin-left: 16px;
+  margin-top: 24px;
+  margin-right: 16px;
+  margin-bottom: 24px;
+}
+
+.description p {
+  color: #344563;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  position: relative;
+  margin-right: 24px;
+  padding-right: 24px;
+}
+
+.source {
+  margin-left: 16px;
+  margin-top: 24px;
+  margin-right: 16px;
+  margin-bottom: 24px;
+  width: 311px;
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+}
+
+.btn1 {
+  width: 134px;
+  height: 48px;
+  padding-left: 12px;
+  padding-right: 12px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  color: #fff;
+  text-align: center;
+  border: none;
+}
+
+.btn2 {
+  width: 159px;
+  height: 48px;
+  padding-left: 12px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  color: #fff;
+  text-align: center;
+  border: none;
+}
+
```

Comment:
- [ ] these button should be created with `<button>` tag not with `<img>` tag please try to fix that